.github/workflows/v3-docs.yml

```
@@ -0,0 +1,42 @@
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: github.event.repository.fork == false
+    steps:
+      - name: Checkout your repository using git
+        uses: actions/checkout@v4
+      - name: Install, build, and upload your site output
+        uses: withastro/action@v2
+        with:
+           path: docs
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 11

(indentation)